core/src/main/resources/hudson/tasks/Fingerprinter/help.html

```diff
@@ -5,19 +5,14 @@
 
   <ul>
     <li>
-      I have <tt>foo.jar</tt> on my HDD but which build number of FOO did it come from?
+      I want to know where a specific <code>foo.jar</code> came from.
     </li>
     <li>
-      My BAR project depends on <tt>foo.jar</tt> from the FOO project.
```

Comment:
This line was deleted without replacement.

Reply:
I don't think that this line adds value.

Reply:
The first example is basically https://ci.jenkins.io/fingerprintCheck, hence the "on my HDD" part.

The second and third examples are for https://ci.jenkins.io/projectRelationship and now that all mention of the "FOO project" is gone, that feature/use case is no longer even hinted at.

Reply:
Wow, that's horrible discovery.

Reply:
Moved this to another branch for another day, it's not directly related to this change